File: source/web/src/types/cliente.ts
export interface Cliente {
  id: string
  codigoCliente: string
  nombre: string
  tipoClienteId: string
  tipoClienteNombre: string
  numeroCuit: string
  numeroIngresosBrutos: string
  erP_Codigo: string
  activo: boolean
}

export interface CreateClienteRequest {
  CodigoCliente: string
  Nombre: string
  TipoClienteId: string
  NumeroCuit?: string
  NumeroIngresosBrutos?: string
  ERP_Codigo?: string
}

export interface UpdateClienteRequest {
  Nombre: string
  TipoClienteId: string
  NumeroCuit?: string
  NumeroIngresosBrutos?: string
  ERP_Codigo?: string
  Activo: boolean
}

export interface TipoCliente {
  codigo: string
  nombre: string
  activo: boolean
}

File: source/api/Meat.Domain/Clientes/Cliente.cs
using Meat.Domain.Enums;
using Meat.Domain.Sucursales;
using Meat.Domain.TiposClientes;
using Meat.Domain.TiposEmpresas;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using Meat.Domain.Empresas;
using Meat.Domain.Shared;

namespace Meat.Domain.Clientes
{
    public class Cliente : ITenantScoped
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.None)]
        public Guid Id { get; set; }
        public string CodigoCliente { get; set; }
        public string Nombre { get; set; }
        public string TipoClienteId { get; set; }
        public TipoCliente TipoCliente { get; set; }
        public string NumeroCuit { get; set; }
        public string NumeroIngresosBrutos { get; set; }
        public bool Activo { get; set; }
        public string ERP_Codigo { get; set; }
        public DateTime FechaActualizacion { get; set; }

        /// <summary>Empresa (tenant) duena del registro. La asigna el MeatContext en el alta.</summary>
        public string EmpresaId { get; set; }
        public virtual Empresa Empresa { get; set; }
    }
}